imports/client/components/ChatPeople.tsx

```diff
@@ -154,6 +163,29 @@ const ChatPeople = (props: ChatPeopleProps) => {
   // or doing signalling).
   const rtcDisabled = useTracker(() => Flags.active('disable.webrtc'), []);
 
+  const recentVoiceActivity = useTracker(() => (
+    CallHistories.findOne({ call: puzzleId })?.lastActivity
+  ), [puzzleId]);
+  const [voiceActivityRelative, setVoiceActivityRelative] = useState<string>();
```

Comment:
This value needs to be in the deps list of the `useLayoutEffect()` below (line 437), since changes in the text here can change wrapping, which can change height, which can cause scroll position changes.